xilem/examples/gallery_label_xilem_view.rs

```diff
@@ -0,0 +1,178 @@
+//! An illustration of the various options for Xilem's Label View (based on Masonry Label Widget)
+// TODOs:
+// add rust code generating each element in a tooltip
+// add the same code in a context menu as a "copy" command
+// add URL support for doc links
+// add non-desktop platforms
+// add to a CI build so you can simply download a binary and use it as a reference
+//   have both debug/release builds so that you can use F11/F12 inspectors?
+//     is a release build useful for basic examples?
+
+#![allow(
+    unused_assignments,
+    reason = "allows NOT having to track section increments {i+=1;}, removing the last one."
+)]
+
+use masonry::dpi::LogicalSize;
+use winit::window::Window;
+use xilem::view::{CrossAxisAlignment, FlexExt, FlexParams};
+#[derive(Default)]
+struct AppState {}
+
+use masonry::core::ArcStr;
+use masonry::parley::fontique;
+use masonry::peniko::Color;
+use winit::error::EventLoopError;
+use xilem::view::{
+    flex, grid, label, label as l, portal, prose, sized_box, Axis, GridExt, Label, Padding, Prose,
+};
+use xilem::{palette::css, EventLoop, FontWeight, LineBreaking, TextAlignment, WidgetView, Xilem};
+const LABEL_COLOR: Color = css::ROYAL_BLUE;
+
+fn title_prose(text: impl Into<ArcStr>) -> Prose {
+    prose(text)
+        .text_size(18.0)
+        .alignment(TextAlignment::Justified)
+        .brush(css::GREEN)
+}
+fn txt_prose(text: impl Into<ArcStr>) -> Prose {
+    prose(text)
+        .text_size(14.0)
+        .alignment(TextAlignment::Start)
+        .brush(Color::from_rgb8(0x11, 0x11, 0x11))
+}
+fn lc(text: impl Into<ArcStr>) -> Label {
+    //colored label
+    label(text).brush(LABEL_COLOR)
+}
+fn app_logic(_d: &mut AppState) -> impl WidgetView<AppState> {
+    let m_c = Color::from_rgb8(0x11, 0x11, 0x11); //main text
+    let l_c = LABEL_COLOR;
+    let mut i = 1;
+
+    portal(
+    flex((
+    (txt_prose("Xilem view::Label formats vGit@25-02 #25b12ad (in a ↕-scrollable area)").text_size(18.0),
+    if cfg!(debug_assertions) {txt_prose(
+     "This is a debug build, so you can use github.com/linebender/xilem/tree/main/masonry#debugging-features:
+        • F11 to toggle a rudimentary widget inspector
+        • F12 to toggle paint widget layout rectangles")
+    } else {txt_prose("This is not a debug build, so F11 widget inspector and F12 widget rectangles tools are not available)\ngithub.com/linebender/xilem/tree/main/masonry#debugging-features")},
+    label(format!("Label: Serif Bold 14 {LABEL_COLOR:?}")).text_size(14.0).weight(FontWeight::BOLD) // float bold=700, FontWeight::parse("normal") for css
+        .font(fontique::GenericFamily::Serif)
+        .alignment(TextAlignment::Start)
+        .brush(l_c) //impl Into<peniko:Brush> brush sets text color, but gradients and images are also supported Solid(color::AlphaColor<Srgb>) Gradient(Gradient) Image(Image),
+        .line_break_mode(LineBreaking::Overflow) //WordWrap Clip Overflow
+        ,
+    title_prose(format!("§{i} .alignment")),{i+=1;},
```

Comment:
Nice use of the section symbol! It's a nice little showcase that we just find the fonts for stuff like that correctly (even though it is a fairly easy case)

Reply:
Does `↕-scrollable area` work for you (in the title)? Or ≝ default signifier in `4 options: ≝Start Middle End Justified`? I see squares there

Reply:
Yes. Both of those resolve for me. The ↕ resolves to an emoji